system/settings/rbp.xml

```diff
@@ -27,12 +27,17 @@
           </control>
           <control type="edit" format="integer" />
         </setting>
+        <setting id="videoplayer.supportmvc" type="boolean" label="38027" help="38028">
```

Comment:
I think this belongs with the other 3D settings under:

 group id="5" label="14232"

As a follow up I will then look at whether the label for that group needs to change as it's currently "Stereoscopic 3D".